### DISCUSSION TITLE
RFC: Changing `@chakra-ui/core` name for v1

### DISCUSSION THREAD

**Discussion by segunadebayo** [chakra-ui/chakra-ui#2086] [General] [answered]
Given that we now have a Vue version of Chakra UI, I've been thinking of the possibility of changing the package name of `@chakra-ui/core` to `@chakra-ui/react`, this makes it clear that we intend to support multiple frameworks in the future:

- `@chakra-ui/react`
- `@chakra-ui/vue`
- `@chakra-ui/react-native`

These are the immediate packages we have on the roadmap hence the need to change the package name.

What do you think?

**Comment by ljosberinn** [comment]
Fits nicely to what we discussed in last weeks meeting too: if we plan on to extract _actual_ **core**, framework-agnostic logic to a separate repo, `@chakra-ui/core` could be repurposed in the long run.

**Comment by chasinhues** [comment]
Absolutely agree, makes sense going forward. Out of curiosity, how would a package with a separate import currently, such as chakra icons, fit into this?

**Reply by segunadebayo** [reply]
Good question. We'll give that some thought.

**Comment by SouravInsights** [comment, marked as answer]
As the Vue package already has the name `@chakra-ui/vue`, it definitely makes sense to have the same for React and React Native.

**Comment by niconiahi** [comment]
Good idea!
